README.md

```diff
@@ -75,36 +70,75 @@ Default is github-pr-check. github-pr-review can use Markdown and add a link to
 
 **Optional**. Tool name to use for reviewdog reporter. Default = `remark-lint`.
 
-## Development
-
-### Release
-
-#### [haya14busa/action-bumpr](https://github.com/haya14busa/action-bumpr)
+## Docker input args
 
-You can bump version on merging Pull Requests with specific labels (bump:major,bump:minor,bump:patch).
-Pushing tag manually by yourself also work.
+Besides the aforementioned input arguments you can also supply additional input arguments for the remark-lint linter using the args keyword [run.args](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#runsargs).
 
-#### [haya14busa/action-update-semver](https://github.com/haya14busa/action-update-semver)
-
-This action updates major/minor release tags on a tag push. e.g. Update v1 and v1.2 tag when released v1.2.3.
-ref: <https://help.github.com/en/articles/about-actions#versioning-your-action>
-
-### Lint - reviewdog integration
-
-This reviewdog action template itself is integrated with reviewdog to run lints
-which is useful for Docker container based actions.
+```yaml
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
```

Comment:
It's not available for this action users, but just for the developer of this action.
Can you remove it or do you think it's useful for users?

Reply:
@haya14busa Thanks a lot for pointing this out I will try to create another way for users to add additional flags.

Reply:
@haya14busa I removed this option from the readme. The user now can supply additional arguments using the `remarks_flags` argument.